Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -4376,6 +4376,8 @@ static bool selectionIsInsideFixedPositionContainer(LocalFrame& frame)
     auto margin = printInfo.margin;
     computePagesForPrintingImpl(frameID, printInfo, pageRects, totalScaleFactor, margin);
 
+    RELEASE_LOG(Printing, "Computing pages for printing (iOS). Page rects size = %zu", pageRects.size());
```

Comment:
`(iOS)` isn't really necessary, since we'll already know the logging is from iOS...